### PR TITLE
Increase the OAuth timeout

### PR DESCRIPTION
**Pull request by hungnguyen94**
Change the logout timeout from 30 min to 2 hours.

### CODE REVIEW

**Comment by trombomart** [comment]
After a lot of discussing I think this is correct :p

File: src/main/java/nl/tudelft/thefirstorder/config/JHipsterProperties.java [hunk 1]
```diff
@@ -185,7 +185,7 @@ public Oauth getOauth() {
 
                 private String secret;
 
-                private int tokenValidityInSeconds = 1800;
+                private int tokenValidityInSeconds = 7200;
```
Comment (cadw) [inline src/main/java/nl/tudelft/thefirstorder/config/JHipsterProperties.java]: Simple fix :P

Comment (rubenwiersma) [inline src/main/java/nl/tudelft/thefirstorder/config/JHipsterProperties.java]: Good addition, now people can use the software properly without having to login all the time.

**Comment by rubenwiersma** [comment]
I'll merge it, tiny change.

**Pull request merged**